### ISSUE TITLE
[Bug]: Aria labels are missing on calendar options

### ISSUE THREAD

**Issue by bh3605** [add2cal/add-to-calendar-button#568]
### What happened?

Tabbing over an option in the pop up after clicking the add to calendar button should read the options in a screen reader.

Would be great if tabbing on the button read the entire content displayed inside of it of it was able to be able to tab inside of the button to read the content on each line

### How to reproduce?

Test for general screen reader accessbility

### Screenshots & more

Location scribbled out. A screen reader should be able to read all of the content inside here
![image](https://github.com/add2cal/add-to-calendar-button/assets/3073436/4d4314e9-08a2-48ce-9e63-61342d13b497)

A screen reader should also be able to read each calendar option when you tab on it.
![image](https://github.com/add2cal/add-to-calendar-button/assets/3073436/ea98a370-b4e5-4dd9-b4d7-fc003656134c)

### System & Environment

Angular, any browser

**Comment by bh3605** [comment]
I found this only applies to Chrome's screen reader extension. Screenreaders like JAWS and NVDA are able to read the content inside of it

**Comment by jekuer** [comment]
This is due to the inability of some setups to access the shadow DOM.
There are some hacky workaround to optimize this. Still, requires some bigger refactoring to not blow up the code.
Put it on the roadmap, but we consider this ticket here as done, since Aria labels are present. Optimizing the shadow DOM access for Screen readers is another topic.
